src/ui/application.vala

```diff
@@ -27,8 +44,14 @@ public class Games.Application : Gtk.Application {
 	}
 
 	public void load_game_list () {
-		var dummy_source = new Games.DesktopGameSource ();
-		dummy_source.each_game ((game) => {
+		if (connection == null)
+			return;
```

Comment:
As for now we have only Tracker as a source it makes sense to have it there, but obviously I don't want a lack of Tracker connection to prevent the other games from being loaded! =)